app/build.gradle.kts

```diff
@@ -18,8 +18,8 @@ android {
         applicationId = "app.lawnchair.lawnicons"
         minSdk = 26
         targetSdk = 31
-        versionCode = 3
-        versionName = "1.2.0"
+        versionCode = 4
+        versionName = "1.3.0"
```

Comment:
Version change should not be included in this PR

Reply:
@Goooler I agree , The reason I added here is , there was some conflict supporting both older version of lawnicons and new version which is this iconpack.

So added a condition in below PR to used old method to apply theme icons untill versioncode 3.

https://github.com/LawnchairLauncher/lawnchair/pull/3116/files#diff-0f635825b22aff4c4eb71e3540b8efd6a93670c6158aa5381177267236e72c11R61

Or atleast the version code should be updated to fully work.